dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/oracle/StatsTaskListGenerator.java

```diff
@@ -61,20 +61,20 @@ class StatsTaskListGenerator {
       ImmutableList.of("hist-cmd-types-statspack", "sql-stats-statspack");
 
   @Nonnull
-  ImmutableList<Task<?>> createTasks(ConnectorArguments arguments) throws IOException {
+  ImmutableList<Task<?>> createTasks(ConnectorArguments arguments, int logDays) throws IOException {
```

Comment:
How does `logDays` differ from `queriedDays` and from `queryLogDays`?

Reply:
Functionally there's no difference, they are different names for the same concept.

Reply:
I would suggest keeping the same name across the codebase to make it easier to read the code without remembering renames when moving across different classes/methods. The rename makes sense, if the meaning of the value changes (e.g. some method is more general). If the meaning doesn't change, keeping the name seems more straight-forward.

Reply:
Done, I replaced `logDays` and `queryLogDays` with `queriedDays`.